### ISSUE TITLE
Get rid of the OutputData struct

### ISSUE THREAD

**Issue by EmilSoleymani** [smiths/vdisp#133]
The `OutputData` struct in the `OutputFormat.jl` module was originally meant to store an instance of `InputData`, which contains all the values it needed for calculations, and to store all the relevant output values. These output value fields were never added, and currently, the `OutputData` only contains an `InputData` instance. Therefore, this struct is redundant. Removing it will require the refactoring of many modules.

**Comment by smiths** [comment]
I agree that we should remove the redundant data structure.  Thank you @EmilSoleymani for creating this issue.
